src/patternfly/components/Pagination/pagination.scss

```diff
@@ -20,7 +20,8 @@
   --pf-c-pagination__nav-page-select--PaddingLeft: var(--pf-global--spacer--md);
   --pf-c-pagination__nav-page-select--PaddingRight: var(--pf-global--spacer--md);
   --pf-c-pagination__nav-page-select--child--MarginRight: var(--pf-global--spacer--xs);
-  --pf-c-pagination__nav-page-select--c-form-control--Width: #{pf-size-prem(48px)}; // custom value to show 2 numbers
+  --pf-c-pagination__nav-page-select--c-form-control--Base: 3ch;
+  --pf-c-pagination__nav-page-select--c-form-control--Chars: 2; // custom value to show 2 numbers
```

Comment:
to avoid a breaking change, and for the end of the var to match the property name the var applies to, can you use this instead?

```
  --pf-c-pagination__nav-page-select--c-form-control--width-base: 3ch;
  --pf-c-pagination__nav-page-select--c-form-control--width-chars: 2;
  --pf-c-pagination__nav-page-select--c-form-control--Width: calc(var(--pf-c-pagination__nav-page-select--c-form-control--width-base) + (var(--pf-c-pagination__nav-page-select--c-form-control--width-chars) * 1ch));
```